e2e-tests/demand-backup-physical/run

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+set -o errexit
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+run_backup() {
+	local storage=$1
+	local backup_name=$2
+
+	cat $test_dir/conf/backup.yml \
+		| $sed -e "s/name:/name: ${backup_name}/" \
+		| $sed -e "s/storageName:/storageName: ${storage}/" \
+		| kubectl_bin apply -f -
+}
+
+run_restore() {
+	local backup_name=$1
+
+	cat $test_dir/conf/restore.yml \
+		| $sed -e "s/name:/name: restore-${backup_name}/" \
+		| $sed -e "s/backupName:/backupName: ${backup_name}/" \
+		| kubectl_bin apply -f -
+}
+
+wait_restore() {
+	local isSharded=${3:-0}
+	local compare_suffix=${2:-"_restore"}
+
+	wait_restore_object "${1}" "requested"
```

Comment:
it is potential problem for this test if your host will have a temporary network issue  (or your laptop just slept ) you will never get 'requested' state